### PR TITLE
[dev-v2.8] Add rancher-turtles chart

### PR DESCRIPTION
**Pull request by salasberryfin**
## Problem

With this change, we're publishing the [rancher-turtles](https://github.com/rancher-sandbox/rancher-turtles) chart to `rancher/charts` for the first time.

## Solution

This just adds the chart and the required references. No other chart should be affected by the change.

### CODE REVIEW

**Comment by recena** [comment]
@salasberryfin [CODEOWNERS](https://github.com/rancher/charts/blob/dev-v2.8/.github/CODEOWNERS) file must be updated.

**Comment by recena** [comment]
@salasberryfin Instead of `WIP` in the PR title, use the draft GH feature.

File: packages/rancher-turtles/rancher-turtles/package.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+url: https://github.com/rancher-sandbox/rancher-turtles/releases/download/v0.5.0/rancher-turtles-0.5.0.tgz
+version: 100.1.0
```
Comment (nicholasSUSE) [inline packages/rancher-turtles/rancher-turtles/package.yaml:2]: Please see the following rules for versioning: https://github.com/rancher/charts/blob/dev-v2.8/README.md#versioning-charts

You are trying to merge at `dev-v2.8` branch.

File: release.yaml [hunk 2]
```diff
@@ -115,3 +115,5 @@ rancher-gatekeeper-crd:
   - 102.1.1+up3.13.0
 neuvector-monitor:
   - 102.0.8+up2.7.3
+rancher-turtles:
+- 100.1.0+up0.5.0
```
Comment (nicholasSUSE) [inline release.yaml:119]: Please see the following rules for versioning: https://github.com/rancher/charts/blob/dev-v2.8/README.md#versioning-charts

You are trying to merge at dev-v2.8 branch.

File: charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher Turtles - the Cluster API operator
+  catalog.cattle.io/kube-version: '>= 1.23.0-0'
+  catalog.cattle.io/namespace: rancher-turtles-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
+  catalog.cattle.io/rancher-version: '>= 2.7.7-0'
```
Comment (nicholasSUSE) [inline charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml:8]: Please se the following Rancher Version Annotations rules at: https://github.com/rancher/charts/blob/dev-v2.8/README.md#rancher-version-annotations

File: charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher Turtles - the Cluster API operator
+  catalog.cattle.io/kube-version: '>= 1.23.0-0'
```
Comment (nicholasSUSE) [inline charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml:4]: Please se the following Rancher Version Annotations rules at: https://github.com/rancher/charts/blob/dev-v2.8/README.md#rancher-version-annotations

File: charts/rancher-turtles/100.1.0+up0.5.0/Chart.lock [hunk 5]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
Comment (nicholasSUSE) [inline charts/rancher-turtles/100.1.0+up0.5.0/Chart.lock:1]: This chart.lock file is weird, I have not seen this before. 
How was this generated? 
What is it used for?

File: charts/rancher-turtles/100.1.0+up0.5.0/charts/cluster-api-operator/charts/cert-manager/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+appVersion: v1.13.2
+description: A Helm chart for cert-manager
+home: https://github.com/cert-manager/cert-manager
+icon: https://raw.githubusercontent.com/cert-manager/cert-manager/d53c0b9270f8cd90d908460d69502694e1838f5f/logo/logo-small.png
+keywords:
+- cert-manager
+- kube-lego
+- letsencrypt
+- tls
+kubeVersion: '>= 1.22.0-0'
```
Comment (nicholasSUSE) [inline charts/rancher-turtles/100.1.0+up0.5.0/charts/cluster-api-operator/charts/cert-manager/Chart.yaml:17]: Please se the following Rancher Version Annotations rules at: https://github.com/rancher/charts/blob/dev-v2.8/README.md#rancher-version-annotations

File: charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+annotations:
```
Comment (nicholasSUSE) [inline charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml:1]: For your referenec this is a good example of Chart.yaml on dev-v2.8

https://github.com/rancher/charts/blob/dev-v2.8/charts/rancher-istio/103.2.0%2Bup1.20.3/Chart.yaml

**Review by nicholasSUSE — CHANGES_REQUESTED** [review]

**Comment by nicholasSUSE** [comment]
@salasberryfin 
Also you need to this process defined here:  https://github.com/rancher/charts/blob/dev-v2.8/README.md#adding-net-new-dependencies-to-dev-27
> When adding new dependencies to dev-2.7, a PR must first be raised and merged in the Rancher repository with the required changes to the pkg/image/origins.go file

If you have this PR ready and merged please attach to the PR description

**Comment by salasberryfin** [comment]
Closing this as we won't be publishing the chart to `rancher/charts` for now.

**Pull request closed without merging**